frontend/src/features/dataset/DatasetHeader.tsx

```diff
@@ -13,11 +13,13 @@ import Button from '../../chrome/Button'
 
 export interface DatasetHeaderProps {
   qriRef: QriRef
+  noBorder?: boolean
   editable?: boolean
 }
 
 const DatasetHeader: React.FC<DatasetHeaderProps> = ({
   qriRef,
+  noBorder,
```

Comment:
provide a default value to ensure this value is always a boolean type within the function body